pkg/querier/distributor_queryable.go

```diff
@@ -134,11 +142,20 @@ func (q *distributorQuerier) Select(ctx context.Context, sortSeries bool, sp *st
 }
 
 func (q *distributorQuerier) streamingSelect(ctx context.Context, sortSeries bool, minT, maxT int64, matchers []*labels.Matcher) storage.SeriesSet {
```

Comment:
It'll be good to expand this for other APIs like label values, label names, series etc. In a dashboard, if the variables don't load, the entire dashboard won't load.

Reply:
Ack!

Reply:
I'm fine if you want to do it in a separate PR.

Reply:
Extended to LabelValuesForLabelName(Stream), LabelNames(Stream) and MetricsForLabelMatchers(Stream) as well